examples/sensors/main.c

```diff
@@ -40,17 +42,28 @@ static void timer_fired(__attribute__ ((unused)) int arg0,
   if (lps25hb)      lps25hb_pressure = lps25hb_get_pressure_sync();
   if (temperature)  temperature_read_sync(&temp);
   if (humidity)     humidity_read_sync(&humi);
-  if (ninedof)      ninedof_read_acceleration_sync(&ninedof_x, &ninedof_y, &ninedof_z);
 
   if (isl29035)       printf("ISL29035:   Light Intensity: %d\n", light);
   if (tmp006)         printf("TMP006:     Temperature:     %d\n", tmp006_temp);
   if (tsl2561)        printf("TSL2561:    Light:           %d lux\n", tsl2561_lux);
   if (lps25hb)        printf("LPS25HB:    Pressure:        %d\n", lps25hb_pressure);
   if (temp)           printf("Temperature:                 %d deg C\n", temp/100);
   if (humi)           printf("Humidity:                    %u%%\n", humi/100);
-  if (ninedof)        printf("FXOS8700CQ: X:               %d\n", ninedof_x);
-  if (ninedof)        printf("FXOS8700CQ: Y:               %d\n", ninedof_y);
-  if (ninedof)        printf("FXOS8700CQ: Z:               %d\n", ninedof_z);
+  if (ninedof)   {
```

Comment:
Hmm.., I see why you restructured this, but I think it would be better to handle all the sensors the same way. I think this app was written with hail/imix in mind, and kind of assumes all the sensors will work, but not that we have more boards with sensors, they really all need the `if .. == TOCK_SUCCESS` treatment.

---

The other issue right now, I think the spacing / alignment on the new ones don't line up (e.g. `Gryo: ` definitely needs more spaces).

Reply:
I fixed the gyro spaces, I think it lines up now. If we remove the ifs, if there is no sensor present, it will just display 0s. Is that ok?

Reply:
I think my question is actually the opposite: should we add more if's, i.e. is the light sensor line just printing 0's on the STM board now?

Reply:
It is not printing anything, as there is no light sensor capsule (or actual sensor). It does not pass the "capsule present" if.

Reply:
OH. Right, I remember how this app works. Sorry.

However, with that in mind, should we expand to `ninedof_accel` `ninedof_mag` and `ninedof_gyro` booleans to match the other sensors?